pages/life-insurance/assess-life-insurance-needs.md

```diff
@@ -1,7 +1,7 @@
 ---
 order: 1
-spoke: More resources
-href: https://www.benefits.va.gov/insurance/lifeins101.asp
-title: Assess Your Life Insurance Needs
+spoke: More Resources
```

Comment:
Heads-up that I think that this needs to be sentence case 👀

Reply:
@ncksllvn Ok, thanks. this looks like a new page that was added to the PR. Whoever set it up probably copied a template from before we sentence-cased. Should I go in and fix this now?